integration_tests/integration/choose-practitioner.spec.ts

```diff
@@ -325,4 +325,80 @@ context('Choose Practitioner', () => {
       choosePractitionerPage.radio('N03F02::OM2').should('not.exist')
     })
   })
+
+  it('should not show team which does not exist in probation estate but is in user preferences', () => {
```

Comment:
I made the calls with the old team code and this is what the workload/ allocations returned